07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,51 @@
+body {
+    font-family: sans-serif;
```

Comment:
You are just a few CSS changes away from completing this assignment with 100% grade. Let me know if you need any assistance.